aten/src/ATen/templates/TypeDefault.cpp

```diff
@@ -29,8 +29,7 @@ namespace TypeDefault {
 
 }  // namespace TypeDefault
 
-namespace {
-auto registerer = torch::import()
+TORCH_LIBRARY(aten, m) {
```

Comment:
I think the invariant that this is a complete declaration of the library is a *big* improvement in quickly grokking what you're looking at.

Reply:
Note that the invariant is not enforced yet